android/BOINC/.idea/misc.xml

```diff
@@ -39,5 +39,5 @@
       </value>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK" />
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="9" project-jdk-type="JavaSDK" />
```

Comment:
Are you sure `project-jdk-name="9"`?

Reply:
Yeah, JDK 9 is required by Robolectric to run tests against Android SDK 29.

The Travis configuration has also been updated to use JDK 9 and therefore Xenial instead of Trusty.